hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -1834,10 +1836,24 @@ public WriteConcurrencyMode getWriteConcurrencyMode() {
     return WriteConcurrencyMode.fromValue(getString(WRITE_CONCURRENCY_MODE));
   }
 
-  public Boolean inlineTableServices() {
+  /**
+   * Are any table services configured to run inline?
+   *
+   * @return True if any table services are configured to run inline, false otherwise.
+   */
+  public Boolean areAnyTableServicesInline() {
```

Comment:
should this check for asyncClean instead of just `isAutoClean()`

Reply:
Yes, both `asyncClean` and `isAutoClean` should be checked here. But this is not used for clean-related logic.

Reply:
Tracked here: HUDI-3686